src/assert.bash

```diff
@@ -278,12 +282,14 @@ assert_output() {
 refute_output() {
```

Comment:
Could you update the function comment too? For example:

```diff
 # Options:
 #   -p, --partial - partial matching
 #   -e, --regexp - extended regular expression matching
+#   - - read unexpected output from the standard input
 # Arguments:
-#   $1 - [=STDIN] unexpected output
+#   $1 - unexpected output
 # Returns:
 #   0 - unexpected matches the actual output
 #   1 - otherwise
```